typings/index.d.ts

```diff
@@ -0,0 +1,28 @@
+declare module 'password-validator' {
+  interface ValidateOptions {
+    list: boolean;
+  }
+
+  class PasswordValidator {
+    validate(password: string, options?: ValidateOptions): boolean | boolean[];
```

Comment:
Shouldn't it be `string[]` here instead of `boolean[]`.
```suggestion
    validate(password: string, options?: ValidateOptions): boolean | string[];
```